packages/react-components/.storybook/preview.js

```diff
@@ -36,7 +36,6 @@ export const parameters = {
     ...rootPreview.parameters.docs,
     container: FluentDocsContainer,
     page: FluentDocsPage,
-    // container: FluentDocsContainer,
```

Comment:
It should not be there 😯

Reply:
what do you mean ? that commented code was there before thus removing it ?

Reply:
Commented code was there and there is no reason to have it there.